app.js

```diff
@@ -72,7 +72,6 @@ for (var i = 0; i < allStores.length; i++) {
 var form = document.getElementById('storeform');
 function inputFormData(event) {
   event.preventDefault();
-  var theform = event.target;
   var newName = event.target.storename.value;
   var newMinCust = Math.floor(event.target.mincust.value);
   var newMaxCust = Math.floor(event.target.maxcust.value);
```

Comment:
I can't comment below this line, but why is your `form.reset()` outside of this function? It would seem to me that when you add new data to the from you'd want to clear out the form, so you'd want `reset` to be called every single time you add new data.